libs/server/repository/infrastructure/src/lib/bitbucket/mappers/map-bitbucket-repository.ts

```diff
@@ -8,6 +8,6 @@ export const mapBitbucketRepository = (repo: BitbucketRepositoryEntity) =>
     displayName: repo.name,
     id: BitbucketUuidUtil.parseFrom(repo.uuid),
     name: repo.slug,
-    owner: repo.owner.username,
+    owner: repo.owner.username || repo.owner.nickname || 'Owner Name Not Found',
```

Comment:
throw an exception if we can't find the owner's name because this value is crucial to request the remote API for the repository.

Reply:
Should be alright now :)